src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -5,10 +5,14 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Http
 {
     public class HttpWorkerOptions
     {
+        public string Type { get; set; } = "http";
+
         public HttpWorkerDescription Description { get; set; }
 
         public WorkerProcessArguments Arguments { get; set; }
 
         public int Port { get; set; }
+
+        public bool EnableHttpRequestForward { get; set; } = true;
```

Comment:
@anthonychu / @yojagad / @fabiocav - Introducing `EnableHttpRequestForward` flag to opt out of proxying http trigger request. Setting this to true by default will make getting started with customHandlers easier. If the user decides to opt-out this flag can be set to `false`

Let me know what you think and also if you have better suggestions for the name.

Note: This flag only applies to functions that only http in and out.

Reply:
Ok makes sense. Will update.